presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java

```diff
@@ -1109,6 +1111,52 @@ public void testViewColumnAccessControl()
         assertAccessAllowed(viewOwnerSession, "DROP VIEW test_invoker_view_column_access");
     }
 
+    @Test
+    public void testViewFunctionAccessControl()
+    {
+        skipTestUnless(supportsViews());
+
+        Session viewOwnerSession = TestingSession.testSessionBuilder()
+                .setIdentity(Identity.ofUser("test_view_access_owner"))
+                .setCatalog(getSession().getCatalog().get())
+                .setSchema(getSession().getSchema().get())
+                .build();
+
+        // TEST FUNCTION PRIVILEGES
```

Comment:
Instead of comments you can extract those to different methods

Reply:
I think these tests requires more love than that: https://github.com/prestosql/presto/issues/3037